client/src/components/Requests/RequestListing/RequestListing.tsx

```diff
@@ -0,0 +1,64 @@
+import { Typography } from '@material-ui/core';
+import Box from '@material-ui/core/Box';
+import Request from '../../../interface/Request';
+import useStyles from './useStyles';
+import Avatar from '@material-ui/core/Avatar';
+import RequestMenu from '../RequestMenu/RequestMenu';
+
+interface Props {
+  request: Request | null | undefined;
+  component: string;
+}
+
+const RequestListing = ({ request, component }: Props): JSX.Element => {
+  const classes = useStyles();
+
+  if (request) {
+    const status =
+      request.accepted === true ? 'accepted' : request.accepted === false ? 'declined' : 'response required';
+
+    const startTime = new Date(request.duration.start)
```

Comment:
Optional: You can move this logic into a function and place it under a utils folder so that it can be used throughout the app :)

Reply:
I will do this if I have spare time. Thank you